server/gql/models/bookings.js

```diff
@@ -0,0 +1,71 @@
+import { GraphQLID, GraphQLInt, GraphQLNonNull, GraphQLObjectType, GraphQLString } from 'graphql';
+import { getNode } from '@gql/node';
+import { createConnection } from 'graphql-sequelize';
+import { timestamps } from './timestamps';
+import db from '@database/models';
+import { totalConnectionFields } from '@utils/index';
+import { sequelizedWhere } from '@database/dbUtils';
+import { addressQueries } from './addresses';
+
+const { nodeInterface } = getNode();
+
+export const bookingFields = {
+  status: { type: GraphQLNonNull(GraphQLString) }
+};
+
+const Booking = new GraphQLObjectType({
+  name: 'Booking',
+  interfaces: [nodeInterface],
+  fields: () => ({
+    ...bookingFields,
+    id: { type: GraphQLNonNull(GraphQLID) },
+    userId: { type: GraphQLInt },
+    cabId: { type: GraphQLInt },
+    email: { type: GraphQLNonNull(GraphQLString) },
+    addressId: { type: GraphQLInt },
+    ...timestamps,
+    addresses: {
```

Comment:
same like addresses, we can add user, cab here, thus we can fetch associated data